configcatclienttests/test_datagovernance.py

```diff
@@ -0,0 +1,427 @@
+import logging
+import unittest
+import requests
+
+from configcatclient import DataGovernance
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+try:
+    from unittest.mock import Mock, ANY
+except ImportError:
+    from mock import Mock, ANY
+
+from configcatclient.configfetcher import ConfigFetcher
+
+logging.basicConfig(level=logging.WARN)
+
+test_json = {"test": "json"}
+
+
+class MockHeader:
+    def __init__(self, etag):
+        self.etag = etag
+
+    def get(self, name):
+        if name == 'Etag':
+            return self.etag
+        return None
+
+
+class MockResponse:
+    def __init__(self, json_data, status_code, etag=None):
+        self.json_data = json_data
+        self.status_code = status_code
+        self.headers = MockHeader(etag)
+
+    def json(self):
+        return self.json_data
+
+    def raise_for_status(self):
+        if 200 <= self.status_code < 300 or self.status_code == 304:
+            return
+        raise Exception(self.status_code)
+
+
+# An organization with Global data_governance config.json representation
+def mocked_requests_get_global(*args, **kwargs):
+    if args[0] == 'https://cdn-global.configcat.com/configuration-files//config_v5.json':
```

Comment:
Would it make sense to move the components of these strings into constants or use the constants from configfetcher?

Reply:
I don't really see the benefit. 
I won't use constants from the configfetcher - as you can easily break the functionality without noticing it by the tests.